zed_test.go

```diff
@@ -236,3 +243,193 @@ func findInputs(t *testing.T, dirs map[string]struct{}, script string, isValidIn
 	}
 	return out, nil
 }
+
+func TestRecordAccessNamed(t *testing.T) {
```

Comment:
I don't think we should move any of these tests in here since it's conventional to put `x.go` tests in `x_test.go`.

(The one move I'd make is to rename `set_test.go` to `complex_test.go`.)

Reply:
This conflicts with the goal of narrowing down the zed package files.  Is there something else we could do to not have the *_test.go files at the top level?